src/text/Text.style.tsx

```diff
@@ -10,7 +10,7 @@ export const StyledText = styled(Text)`
   }
 
   &.kirk-text-button {
```

Comment:
`TextDisplayType.BUTTON` is used nowhere and is more logic to be `color.blue`

Reply:
If I'm not mistaked a lot of this can be cleaned to use typography components instead @gabrielagabriel  ?

Reply:
In the end no need to use and update the legacy `TextDisplayType.BUTTON`.
I directly use `rightBodyColor` from the `Item` component.